compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -1511,7 +1563,7 @@ static void buildStringCastFunction(EnumType* et) {
 
 
 void buildDefaultDestructor(AggregateType* ct) {
-  if (function_exists("deinit", 2, dtMethodToken, ct))
+  if (function_exists("deinit", dtMethodToken, ct))
```

Comment:
Moved to PR #6168. I will continue to test these together but can merge that one first.